libs/coin-modules/coin-polkadot/src/network/index.ts

```diff
@@ -16,6 +16,7 @@ import {
 } from "./sidecar";
 import BigNumber from "bignumber.js";
 import { PolkadotAccount, PolkadotNomination, PolkadotUnlocking, Transaction } from "../types";
+import network from "@ledgerhq/live-network/network";
```

Comment:
Personal opinion but I wouldn't use live-netwok in coin modules.

Reply:
Live-Network is our (Ledger Live) de facto standard library for network request.